inc/sainexthop.h

```diff
@@ -226,6 +226,33 @@ typedef enum _sai_next_hop_attr_t
      */
     SAI_NEXT_HOP_ATTR_QOS_TC_AND_COLOR_TO_MPLS_EXP_MAP,
 
+    /**
+     * @brief To enable/disable source MAC rewrite
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_NEXT_HOP_ATTR_DISABLE_SRC_MAC_REWRITE,
```

Comment:
Couple of questions here:
-  Is the idea here to perform L3 routing but not rewrite the packet ?
- Would it be possible to update the PR description with the usecase ?

Reply:
yes we rely on L3 based IP forwarding, and have some scenarios for passing through without rewrites. I have updated the PR description.